File: backend/calApp/calorie-service/src/main/java/com/caliq/calorie_service/models/dto/UserInfoResponse.java
package com.caliq.calorie_service.models.dto;
import java.math.BigDecimal;
import java.time.LocalDate;
public record UserInfoResponse(BigDecimal weight,
                               BigDecimal height,
                               String sex,
                               String activity_type,
                               LocalDate birthday_date,
                               String goalType) {
}

File: androidApp/NytriSync/app/src/main/java/com/ga3t/nytrisync/data/model/UserInfoResponse.kt
package com.ga3t.nytrisync.data.model

import java.math.BigDecimal

data class UserInfoResponse(
    val weight: BigDecimal,
    val height: BigDecimal,
    val sex: String,
    val activity_type: String,
    val birthday_date: String,
    val goalType: String
)